.prettierignore

```diff
@@ -6,6 +6,5 @@ CHANGELOG.md
 README.md
 .github/ISSUE_TEMPLATE
 site/docs-manifest.json
-.changeset
```

Comment:
I saw this was added [ages ago](https://github.com/seek-oss/vanilla-extract/pull/132), but no context other than "WIP".

Reply:
I think we added it because it often prevents you from modifying them from the github UI, which we do frequently. Also there's not much point formatting them as they're removed and formatted via `Version Packages` anyway.